test/validateRules.js

```diff
@@ -68,17 +73,21 @@ const ajv = new Ajv({ loadSchema });
   ruleList.data.forEach((rule) => {
```

Comment:
FWIW, you can also define the loop index, `i`, in [`Array#forEach`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/forEach) and JavaScript will tell you the index (versus tracking it separately and having to increment it on L86 below).

```js
ruleList.data.forEach((rule, i) => {
```

Reply:
Wow, how did I not know this. Thanks!